### DISCUSSION TITLE
post/asn-1-asn-registration/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#25] [Announcements]
# post/asn-1-asn-registration/

前言AS & BGP在计算机网络的课程中, 我们都知道当今的 Internet 实际上是由多个自治系统 (Autonomous system) 通过 BGP (边界网关协议) 相互连接形成的. 每个 AS 有一个编号, 被称为 ASN. 每个 AS 里会包含一些 IP 段, 而每个 IP 也必须在某个 AS 内被宣告才能接入互联网. 例如南京大学在校内部分网络可能被分配 114.212.0

https://blog.lyc8503.net/post/asn-1-asn-registration/

**Comment by liuxyon** [comment]
还能便宜赞助ASN不？找谁联系？

**Comment by xiaoman1221** [comment]
您好，我想问一下发票的问题。
我选择的LIR提供商是：Lagrange Cloud
我选择的主机商是：Vultr
我的问题是：
The new invoice provided is also addressed to you personally rather than the company, and also does not state the location of the service.
想问一下博主使用的是哪家主机商，它需要我提供公司的发票，而且发票上需要有地区等信息

**Reply by lyc8503** [reply]
我用的 LIR 没有这个要求。我用的 VPS 是 BuyVM，不过我猜大部分主机商的地址和信息可以随便填。

**Reply by xiaoman1221** [reply]
谢谢您的恢复

**Comment by NannaOlympicBroadcast** [comment]
学长告诉我你自己的自治ipv4范围吗
